src/scss/irlpodcast.scss

```diff
@@ -626,6 +626,48 @@ a.episode-shownotes-link {
     }
 }
 
+.next {
+    &:before {
+        @include gradient-bar();
+    }
+}
+
+.next-next,
+.next-prev {
+    box-sizing: border-box;
+    display: block;
+    padding-top: 20px;
+
+    span {
+        display: block;
+    }
+
+    a {
+        @include font-size-level3;
+    }
+}
+
+.next-next {
+    text-align: right;
+}
+
+@media #{$mq-phone-wide} {
+    .next-next,
+    .next-prev {
+        float: left;
+    }
+
+    .next-prev {
+        padding-right: 20px;
+    }
+
+    .next-next {
+        float: right;
+        padding-left: 20px;
```

Comment:
I think the UI could use a little attention for skinnier viewports.

When `< $mq-phone-wide`, the left/right alignment looks a little broken.

<img width="514" alt="screen shot 2018-08-28 at 3 12 13 pm" src="https://user-images.githubusercontent.com/317532/44748245-e6fe1500-aad4-11e8-9dbd-712a1ef16a11.png">

When `>= $mq-phone-wide`, the alignment is also a little broken if the titles are long-ish.

<img width="684" alt="screen shot 2018-08-28 at 3 13 58 pm" src="https://user-images.githubusercontent.com/317532/44748286-072dd400-aad5-11e8-9ec8-cae5385b27a5.png">

Maybe just left align them until `$mq-phone-wide` (or `$mq-tablet`?) and then set a max width when floating?